deploy/StableJoeStaking.js

```diff
@@ -12,6 +17,7 @@ module.exports = async function ({ getNamedAccounts, deployments }) {
     rewardToken = "0x9Ad6C38BE94206cA50bb0d90783181662f0Cfa10";
     feeCollector = "0x2fbB61a10B96254900C03F1644E9e1d2f5E76DD2";
     depositFeePercent = 0;
+    smolJoes = "0x0000000000000000000000000000000000000000"; // placeholder for now
```

Comment:
This will revert on initialize, use any ERC20 contract as they also define `balanceOf` function or use a proper mock NFT contract